src/shogun/statistical_testing/MMD.h

```diff
@@ -223,6 +223,16 @@ class CMMD : public CTwoSampleTest
 	index_t get_num_null_samples() const;
 
 	/**
+	 * Method to sets random seed for CrossValidation.
+	 *
+	 * @param p_seed seed for random generator.
+	 */
+	void set_random_seed(int32_t p_seed);
```

Comment:
yeah, you are right. so that's the problem I have when I'm working on the global random removal issue. Usually when we want to ensure all the module we call will have a fixed seed to ensure they have same output every time, we call **sg_rand->set_seed(seed)** But after we remove that:

- Do we want a fix seed for every module we used in the tests?(I guess yes :)

- How to pass a fix seed to nested module? like when we do kernel selection, we need CrossValidationMMD inside, is that necessary to change some places just to ensure we unit test can passed every time?

- Maybe we have a better way to deal with these unit tests to make sure we get same result every time rather than give a fixed seed?

Maybe I will have more questions about this, but please give me some guides above. Thanks a lot
@karlnapf @lambday